src/cluster_legacy.h

```diff
@@ -218,6 +224,24 @@ union clusterMsgData {
     } module;
 };
 
+/* Light message data and it's supported message types. */
+typedef struct {
+    uint32_t message_len;
+    unsigned char message_data[8]; /* 8 bytes just as placeholder. */
```

Comment:
I think we should use 4 bytes
```suggestion
    char message_data[4]; /* 4 bytes just as placeholder. */
```

Reply:
We have removed the Multi publish struct for this PR